src/transforms/transform-pull-request.ts

```diff
@@ -157,3 +159,88 @@ const getBranches = async (gitHubInstallationClient: GitHubInstallationClient, p
 		}
 	];
 };
+
+// TODO: TYPES N THINGS - reviews type and return type?
+// this will default the current webhook oh no
+export const transformPullRequestNew = (_jiraHost: string, pullRequest: pullRequestNode, reviews?: any, log?: Logger) => {
+	const issueKeys = extractIssueKeysFromPr(pullRequest);
+
+	if (isEmpty(issueKeys) || !pullRequest.headRef.repository) {
+		log?.info({
+			pullRequestNumber: pullRequest.number,
+			pullRequestId: pullRequest.id
+		}, "Ignoring pullrequest since it has no issue keys or repo");
+		return undefined;
+	}
+
+	try {
+		return {
+			author: getJiraAuthor(pullRequest.author),
+			commentCount: pullRequest.comments.totalCount || 0,
+			destinationBranch: pullRequest.baseRef?.name || "",
+			destinationBranchUrl: `https://github.com/${pullRequest.baseRef?.repository?.owner?.login}/${pullRequest.baseRef?.repository?.name}/tree/${pullRequest.baseRef?.name}`,
+			displayId: `#${pullRequest.number}`,
+			id: pullRequest.number,
+			issueKeys,
+			lastUpdate: pullRequest.updatedAt,
+			reviewers: mapReviews(reviews.nodes),
+			sourceBranch: pullRequest.headRef?.name || "",
+			sourceBranchUrl: `https://github.com/${pullRequest.headRef?.repository?.owner?.login}/${pullRequest.headRef?.repository?.name}/tree/${pullRequest.headRef?.name}`,
+			status: pullRequest.state, // test closed and declined behaviou// mapStatus(pullRequest.state, pullRequest.merged_at), mapStatus(pullRequest.state, pullRequest.mergedAt),
+			timestamp: pullRequest.updatedAt,
+			title: pullRequest.title,
+			url: pullRequest.url,
+			updateSequenceId: Date.now()
+		};
+	} catch (err) {
+		throw new Error();
+	}
+};
+
+// TODO: define arguments and return
+// todo types
+const mapReviews = (reviews: any = []): any[] => {
```

Comment:
I hate this whole funciton, its basically a clone of the existing one with a few tweaks on the attributes to match the new object....... decided against refactoring at this point, it works and don't want to introduce more change